CMakeLists.txt

```diff
@@ -165,6 +165,9 @@ if( TRACCC_SETUP_ALPAKA )
    else()
       add_subdirectory( extern/alpaka )
    endif()
+
+   # Include the Alpaka CMake code.
+   list( APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/extern/alpaka/cmake" )
```

Comment:
Wait, wait, wait...

The `traccc-alpaka-functions.cmake` file should be put under `cmake/`. Along with all of our other such CMake files. No reason to separate it like this. 🤔

And as long as you put it under `cmake/`, you also don't need to do anything to make it visible to any sub-directory of the project.